desidiff/nb/desidiff.ipynb

```diff
@@ -167,9 +178,25 @@
     "            print(tid)\n",
     "            for b in difflux.keys():\n",
     "                w=numpy.where(difmask[b] ==0)[0]\n",
-    "                plt.plot(difwave[b][w],difflux[b][w],color='red')\n",
-    "                plt.plot(newwave[b][w],newflux[b][w],color='blue',alpha=0.5)\n",
+    "#                 plt.plot(difwave[b][w],difflux[b][w],color='red')\n",
+    "#                 plt.plot(newwave[b][w],newflux[b][w],color='blue',alpha=0.5)\n",
     "                \n",
+    "                if b == 'b':\n",
```

Comment:
Sometimes there are spectrograph failuers.  So there should not be hard wiring of bands that may not exist.

Reply:
Sure, I can make it to be whatever the last band would be